resources/assets/less/bem/beatmap-scoreboard-table.less

```diff
@@ -164,6 +170,11 @@
 
   &__user-link {
     color: #555;
+
+    &:hover {
+      font-weight: bold;
```

Comment:
this moves the columns when hovering on the longer name in the table when the table is at minimum size.